### PR TITLE
feat: homepage styles

### LINKED ISSUE #8: Homepage Styling
- Implement dark vs light styling using section metadata (see example on German `index.docx`)
- All images have a in/out transition
- Margins/Paddings
- Font sizes
- Colors
- Remove auto-transform of links on entire site; style links on homepage as buttons

### PR DESCRIPTION
**Pull request by cuzuco2**
Changes for the dak-light style on pages

Fix #8

Test URLs:
- Before: https://main--elco-aerotop-sg--hlxsites.hlx.page/de/
- After: https://dark-light-style--elco-aerotop-sg--hlxsites.hlx.page/de/

### CODE REVIEW

**Comment by cuzuco2** [comment]
@ekremney @solaris007 Please review

**Comment by solaris007** [comment]
please fix linting error:
```
styles/styles.css
 175:1  ✖  Expected selector "a:any-link" to come before selector "body       no-descending-specificity
           .dark a"
 180:1  ✖  Expected selector "a:hover" to come before selector "body .dark    no-descending-specificity
           a"
```

File: styles/styles.css [hunk 1]
```diff
@@ -120,6 +120,15 @@ body.appear {
   display: unset;
 }
```
Comment (solaris007) [inline styles/styles.css]: please fix linting error:
```
styles/styles.css
 175:1  ✖  Expected selector "a:any-link" to come before selector "body       no-descending-specificity
           .dark a"
 180:1  ✖  Expected selector "a:hover" to come before selector "body .dark    no-descending-specificity
           a"
```

**Comment by solaris007** [comment]
The top of the dark area is too low. Probably a general issue on all pages. Please fix as part of this PR.

Screenshot 2023-01-17 at 08 09 45

**Comment by solaris007** [comment]
@cuzuco2 i allowed myself to include a couple of additional homepage-related things in the linked issue #8.

- Implement dark vs light styling using section metadata (see example on German `index.docx`)
- All images have a in/out transition (something like: `    transform: matrix3d(1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1);`)
- Margins/Paddings
- Font sizes
- Colors
- Remove auto-transform of links on entire site; style links on homepage as buttons

File: styles/styles.css [hunk 2]
```diff
@@ -120,6 +120,15 @@ body.appear {
...
 
+body .dark {
```
Comment (solaris007) [inline styles/styles.css]: The homepage content, mostly in the dark areas seems to center content. Can you investigate?

Comment (cuzuco2) [reply styles/styles.css]: @solaris007 sometimes it is centered sometimes it is not:
image
image
I understand that we cannot add just a metadata for such thing, how would you like to approach this? with a new block?

File: styles/styles.css (hunk 1 above)
Comment (cuzuco2) [reply styles/styles.css]: done

File: styles/styles.css (hunk 2 above)
Comment (cuzuco2) [reply styles/styles.css]: Done, I added a new metadata to the section.

**Comment by cuzuco2** [comment]
@solaris007 @ekremney Please review now, it should address all the different elements except for the image transition which has it's own separate issue.
Also this doesn't deal with image floating for now, or images with columns with title and descriptions as seen in here:
image

File: styles/styles.css [hunk 3]
```diff
@@ -338,3 +337,68 @@ main .section[data-section-status='initialized'] {
...
+  white-space: nowrap; 
+}
+
+.section.dark {
+  background-color: var(--secondary-background-color);
+  color: var(--secondary-text-color);
+  padding-bottom: 65px;    
+}
+
+.section.dark.center {
```
Comment (ekremney) [inline styles/styles.css:380]: _nitpicking_: please fix the indentation for this and the following blocks

**Review by ekremney — DISMISSED** [review]

File: styles/styles.css [hunk 4]
```diff
@@ -338,3 +337,68 @@ main .section[data-section-status='initialized'] {
 main .section.highlight {
   background-color: var(--highlight-background-color);
 }
+
+body > main > div> div {
```
Comment (solaris007) [inline styles/styles.css]: i would be more specific here and apply to `section`

File: styles/styles.css [hunk 5]
```diff
@@ -338,3 +337,68 @@ main .section[data-section-status='initialized'] {
...
+  padding-top: 45px;
+}
+
+body > main > div:last-child {
+  margin-bottom: 0;
+}
+
+body > main > div:first-child > div {
```
Comment (solaris007) [inline styles/styles.css:349]: apply to first `section`

**Review by solaris007 — DISMISSED** [review]

**Comment by solaris007** [comment]
will you handle this item of issue #8 separately?

All images have a in/out transition (something like:     transform: matrix3d(1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1);)

File: styles/styles.css (hunk 3 above)
Comment (cuzuco2) [reply styles/styles.css:380]: done

File: styles/styles.css (hunk 4 above)
Comment (cuzuco2) [reply styles/styles.css]: done

**Comment by cuzuco2** [comment]
@solaris007 , @ekremney  changed the code according to your review. I will handle the transition separately as the transition has it's own issue so like we said we will create a different branch and different PR for each different issue.
@solaris, if it is fine with you please merge the pull request

**Review by solaris007 — APPROVED** [review]

**Review by ekremney — APPROVED** [review]

**Pull request merged**